app/adapters/application.js

```diff
@@ -1,9 +1,6 @@
 import DS from 'ember-data';
 
 export default DS.JSONAPIAdapter.extend({
-  host: 'https://beta.gouv.fr',
```

Comment:
We should merge public/startup.json to the actual response to handle **only** special cases, not all of them.